### PR TITLE
refactor names of functionaries

### PR DESCRIPTION
**Pull request by akanex115**
# Background
issue https://github.com/ajktown/wordnote/issues/44
We refactored the function names in the PRs(https://github.com/ajktown/wordnote/pull/158 & https://github.com/ajktown/wordnote/pull/163), but there were components that were not changed.

## TODOs
- [x]  Refactor every hook to have the prefix on for functions they return
- [x]  Refactor every component that takes the function from hook to keep the same name

## Checklist (Developers)
- [x] Make this PR as a draft first
- [x] Title is checked
- [x] Background & TODOs are filled
- [x] Assignee is set
- [x] Labels are set
- [x] Project is created & linked, if multiple PRs are associated to this PR
- [x] Appropriate issue(s) is(are) linked to this PR under `development`
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
Please copy and paste the following into your review, ensuring each checkbox is marked as checked
```
- [ ] Check if there are any other missing TODOs that are not yet listed
- [ ] Review Code
- [ ] Every item on the checklist has been addressed accordingly
- [ ] If `development` is associated to this PR, you must check if every TODOs are handled
```

### CODE REVIEW

**Review by mlajkim — COMMENTED** [review]
nice. lgtm at this point.

**Comment by akanex115** [comment]
I checked all the hooks and the components that use the function returned by the hooks.

File: src/hooks/use-dynamic-focus.hook.ts [hunk 1]
```diff
@@ -21,10 +21,10 @@ export const useDynamicFocus = (
     }, milliSecondsWait)
   }, [ref, milliSecondsWait])
 
-  const onDynamicFocus = useCallback(async () => {
+  const onGetDynamicFocus = useCallback(async () => {
```
Comment (mlajkim) [inline src/hooks/use-dynamic-focus.hook.ts:24]: since this does not `get` from the API, it should stay `onDynamicFocus`

**Review by mlajkim — COMMENTED** [review]
First review

File: src/hooks/use-dynamic-focus.hook.ts (hunk 1 above)
Comment (akanex115) [reply src/hooks/use-dynamic-focus.hook.ts]: Handled in [3169ef1](https://github.com/ajktown/wordnote/pull/167/commits/3169ef1e95da99a8c800ed3259196109ac574f6b)

**Review by mlajkim — APPROVED** [review]
OK

**Comment by mlajkim** [comment]
- [x] Check if there are any other missing TODOs that are not yet listed
- [x] Review Code
- [x] Every item on the checklist has been addressed accordingly
- [x] If `development` is associated to this PR, you must check if every TODOs are handled

**Pull request merged**